src/OpenConext/ProfileBundle/Resources/views/layout.html.twig

```diff
@@ -2,15 +2,9 @@
 
 {% block stylesheets %}
     <!--[if IE]>
-    <script src="{{ asset('js/es5-shim.min.js') }}"></script>
-    <script src="{{ asset('js/classList.min.js') }}"></script>
-    <script src="{{ asset('js/selectivizr.min.js') }}"></script>
+    <script src="{{ asset('build/app.js') }}"></script>
```

Comment:
Will we run into issues on IE if we include build/app.js twice?

Reply:
Ah, yes this is a leftover of the IE9 support we dropped. Removing this!